### ISSUE TITLE
Any chances of adding some kind of post-processing?

### ISSUE THREAD

**Issue by XobSod** [Huachao/vscode-restclient#503]
Hi,
On the beginning I just want to say that this is one great tool you have here ;)
I have question, have you thought about adding some kind of post-processing mechanism? I am using this tool for debugging rest service, and to do this I have to send request and read response, but I am not interested in whole response, but particular part of it, sometimes even one line of it. And each time I have to manually search for the section I am fixing right now.
It would be great to have possibility of defining a post-process routine that will allow me to forward response to lets say grep, to filter out the line I am interested in, or in general to script that will display the part of response I have to search every time I communicate  with service. It would be really nice thing to have.

**Comment by tupini07** [comment]
I also agree it would be great to be able to apply arbitrary modifications on a response. For instance, I sometimes find myself in the need of getting some data from an endpoint, encode it with base64, and then forward it to another endpoint.

**Comment by ChayimFriedman2** [comment]
Duplicate of #521 but the other is more general.

**Comment by XobSod** [comment]
Any update on either of tickets?
